spec/components/courses/international_students_component_spec.rb

```diff
@@ -13,15 +13,20 @@
         funding_type: 'fee',
         provider: provider,
       )
-      result = render_inline(described_class.new(course: CourseDecorator.new(course)))
+      @result = render_inline(described_class.new(course: CourseDecorator.new(course)))
+    end
+
+    it 'tells candidates they’ll need the right to study' do
+      expect(@result.text).to include('You’ll need the right to study in the UK')
```

Comment:
Instead of the instance variable, by default, you have access to `page` so `page.text`. But to make it more readable at the top of the file under the describe, you could declare `alias_method :component, :page` allowing you to write `component.text` or even `result.text` if you prefer

Reply:
oh yes, thats a much better idea 👍